tesseract/tesseract_planning/tesseract_motion_planners/CMakeLists.txt

```diff
@@ -78,6 +77,9 @@ target_include_directories(${PROJECT_NAME}_ompl PUBLIC
     "$<INSTALL_INTERFACE:include>")
 target_include_directories(${PROJECT_NAME}_ompl SYSTEM PUBLIC
     ${OMPL_INCLUDE_DIRS})
+if(OMPL_VERSION VERSION_LESS "1.4.0")
+    target_compile_definitions(${PROJECT_NAME}_ompl PUBLIC OMPL_LESS_1_4_0=ON)
+endif()
```

Comment:
Could you enable the TESSERACT_ENABLE_TESTING flag again? Somewhere it got commented out so tests are always run

Reply:
fixed.